src/components/top-bar/top-bar.stories.tsx

```diff
@@ -39,30 +43,27 @@ export const StylesOnlyUsage: StoryObj<typeof TopBar> = {
     return (
       <nav>
         <ElTopBar>
+          <ElTopBarLogo>
+            <ReapitLogoIcon className={elReapitLogo} />
+          </ElTopBarLogo>
           <ElTopBarMainNav>
             <ElNavItemAnchor href="#">
               <ElNavItemLabelContainer>Button 1</ElNavItemLabelContainer>
             </ElNavItemAnchor>
             <ElNavItemAnchor href="#">
               <ElNavItemLabelContainer>Button 2</ElNavItemLabelContainer>
             </ElNavItemAnchor>
-            <ElNavItemAnchor href="#">
-              <ElNavItemLabelContainer>Button 3</ElNavItemLabelContainer>
-            </ElNavItemAnchor>
-            <ElNavItemAnchor href="#">
-              <ElNavItemLabelContainer>Button 4</ElNavItemLabelContainer>
-            </ElNavItemAnchor>
             <ElMenu>
               <ElNavDropdownButton aria-haspopup="true" aria-expanded="true" role="button" type="button">
                 More
                 <Icon icon={'chevronUp'} intent="default" fontSize="1rem" />
               </ElNavDropdownButton>
               <ElMenuPopover className={elTopBarMenuPopover}>
                 <ElMenuList>
-                  <ElMenuItemButton role="menuitem">Button 5</ElMenuItemButton>
-                  <ElMenuItemButton role="menuitem">Button 6</ElMenuItemButton>
+                  <ElMenuItemButton role="menuitem">Button 3</ElMenuItemButton>
+                  <ElMenuItemButton role="menuitem">Button 4</ElMenuItemButton>
                   <ElMenuItemAnchor href="/#" role="menuitem">
-                    Button 7
+                    Button 5
                   </ElMenuItemAnchor>
```

Comment:
need to reduce the amount of button here, so it looks good in the mdx docs, since mdx docs display the preview in smaller container, while stories has full page.